massgov/mayflower
# DP-13660: Update homepage mobile styles

## Description
Made image credit use relative positioning on small screens to prevent
popular search terms with long titles from overlapping with the image
credit text and map marker.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-13660)
- [Drupal PR](https://github.com/massgov/mass/pull/3479)

## Steps to Test

1. Visit the following URL: https://mayflower.digital.mass.gov/b/patternlab/DP-13660--homepage-styling/?p=organisms-search-banner
2. Note that the search banner appears as it always has (no overlapping of text on the image credit).

## Additional Notes:

This PR works in tandem with a Drupal PR, but they do not necessarily need to be released at the same time. However, it would be best for the original ticket if they were.

#### Impacted Areas in Application


* Page > Homepage
* Organisms > Search Banner
